### PR TITLE
Forbidden test page

### PR DESCRIPTION
**Pull request by fmarier**
Add a new test page for this new type of Safe Browsing blocking: https://bugzilla.mozilla.org/show_bug.cgi?id=1216723

### CODE REVIEW

**Comment by fmarier** [comment]
I asked Matej to review the copy on https://bugzilla.mozilla.org/show_bug.cgi?id=1216727

**Comment by fmarier** [comment]
The copy was just [approved](https://bugzilla.mozilla.org/show_bug.cgi?id=1216727#c2) by Matej.

@bensternthal, can you take a look when you have a minute?

File: firefox/forbidden.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        <div>
+		<h1><a href="http://www.mozilla.org/firefox/" title="Back to home page">Mozilla Firefox</a></h1>
+		<a href="http://www.mozilla.org/" class="mozilla">visit <span>mozilla</span></a>
+
+        </div>
+    </div>
+    <!-- end #header -->
+
+
+<div id="main-feature">
```
Comment (bensternthal) [inline firefox/forbidden.html]: Nit but important.. can you tidy the indentation throughout the file.

Comment (fmarier) [inline firefox/forbidden.html]: > Nit but important.. can you tidy the indentation throughout the file.

I've just run the whole file through `M-x indent-region`.

**Comment by bensternthal** [comment]
Looks good, checked out locally and no issues .

**Pull request merged**